src/_pytest/tmpdir.py

```diff
@@ -131,8 +131,8 @@ def getbasetemp(self) -> Path:
             # rootdir's permissions. Historically 0o755 was used, so we can't
             # just error out on this, at least for a while.
             if hasattr(os, "getuid"):
+                uid = os.getuid()  # type:ignore[attr-defined]
```

Comment:
This was done in a separate commit (84c5fe4d440faa70b69e618779828af55436b569).

Reply:
I think the `hasattr` check should be `if sys.platform != 'win32'`  -- this matches the guard from typeshed: https://github.com/python/typeshed/blob/48a346920bbc24165bd9e9cf29a2b5140eae2abc/stdlib/os/__init__.pyi#L388

Reply:
Agree, thanks. Updated. 👍